systers_portal/templates/meetup/add_meetup.html

```diff
@@ -0,0 +1,42 @@
+{% extends "meetup/base.html" %}
+
+{% block title %}
+  Add meetup to {{ meetup_location.name }}
+{% endblock %}
+
+{% block head %}
+<link  href="/static/css/jquery-ui.min-1.11.4.css" rel="stylesheet" type="text/css"/>
+  <link  href="/static/css/jquery-ui-timepicker.0.3.3.css" rel="stylesheet" type="text/css"/>
+{% endblock %}
+
+{% load crispy_forms_tags %}
+{% block meetup_location_page_content %}
+
+  <div class="mt20 mb40"></div>
+  <div class="row ml20">
+    <div class="col-md-12">
+      <h1>Add meetup to {{ meetup_location.name }}</h1>
+      <hr/>
+    </div>
+    <div class="col-md-8">
+      <div class="well">
+        {{ form.media }}
+        {% crispy form %}
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block scripts %}
+<script src="/static/js/libs/jquery-ui.min-1.11.4.js"></script>
+<script src="/static/js/libs/jquery-ui-timepicker-0.3.3.js"></script>
+  <script type="text/javascript">
```

Comment:
It's always better to separate JS into a file and then include it.
Also regarding the JS code itself. You are doing `$(function() {});` also called "jquery ready" twice. You need to do it only once. You probably need to read a bit more about the concept and what it does.

Another issue that I have experienced is the inability to actually select a date via the datepicker. Short look at it gave me a hint that we are selecting the date in the format dd/mm/yyyy and the picker expects mm/dd/yyyy.

![image](https://cloud.githubusercontent.com/assets/2039122/9154841/46f8164a-3e9a-11e5-8c37-16e48b46cbdf.png)

Also here's how the time select looks like to me (probably related to the missing images shown in the console).
![image](https://cloud.githubusercontent.com/assets/2039122/9154851/8cf8c32e-3e9a-11e5-94f4-1883e22f88ba.png)

May I suggest a nicer **datepicker**? http://amsul.ca/pickadate.js/ is a bootstrap looking time and date picker. I've used it recently and was very satisfied with the result.